src/test/java/com/google/api/codegen/testdata/py/python_main_library.baseline

```diff
@@ -420,27 +421,27 @@ class LibraryServiceClient(object):
         RPC method comment may include special characters: <>&\"``'@.
 
         Example:
-          >>> from google.cloud.example import library_v1
-          >>>
-          >>> client = library_v1.LibraryServiceClient()
-          >>>
-          >>> shelf = {}
-          >>>
-          >>> response = client.create_shelf(shelf)
+            >>> from google.cloud.example import library_v1
+            >>>
+            >>> client = library_v1.LibraryServiceClient()
+            >>>
+            >>> shelf = {}
+            >>>
+            >>> response = client.create_shelf(shelf)
 
         Args:
-          shelf (Union[dict|:class:`google.cloud.proto.example.library.v1.library_pb2.Shelf`]): The shelf to create.
-            If a dict is provided, it must be of the same form as the protobuf
-            message :class:`google.cloud.proto.example.library.v1.library_pb2.Shelf`
-          options (:class:`google.gax.CallOptions`): Overrides the default
-            settings for this call, e.g, timeout, retries etc.
+            shelf (Union[dict, ~.library_pb2.Shelf]): The shelf to create.
```

Comment:
I just realized I missed something here before (my bad!) -- this probably should be `~google.cloud.library_v1.types.Shelf`, because that is where the docs will find it (the link will not work with the _pb2).

Also, in the narrative description, where it says "message ~.library_pb2.Shelf", can that just say "message Shelf"?

Reply:
I made all narrative descriptions (return or argument) to be of the form 
```
:class:`~google.cloud.library_v1.types.Shelf`
```